documentation/algorithms/replica_exchange.md

```diff
@@ -1,15 +1,42 @@
 # Replica Exchange: the main ingredient of Chainsail
 
-Replica Exchange is the Markov Chain Monte Carlo (MCMC) algorithm at the heart of Chainsail and it's main feature. 
+## General idea
+Replica Exchange is the Markov Chain Monte Carlo (MCMC) algorithm at the heart of Chainsail and its main feature. 
 In a nutshell, Replica Exchange works by sampling a series of increasingly "flatter" versions (_replicas_) of a probability distribution with a "local" MCMC algorithm such as [Hamiltonian Monte Carlo](./hmc.md) and occasionally exchanging states between all those separate Markov chains.
 That way, the Markov chain that samples the distribution of interest can escape from modes it is otherwise likely to be trapped in.  
+
+## Tempered ("flattened") distributions
 Replica Exchange requires the choice of a series of probability distributions, which interpolate between the target ("cold") distribution and some very easy to sample, "flat" ("hot") distribution.
 It is convenient to choose a parameterized family of "tempering" distributions and then vary its parameter to set a _schedule_.
-While that family of tempering distributions is essentially arbitrary, Chainsail currently implements only the most traditional and popular one, where `p(x|beta)=p(x)^beta` with `0 < beta <= 1` and `p(x)` your probability distribution of interest.
-`beta` is often referred to as an (inverse) temperature in analogy to the [canonical ensemble](https://en.wikipedia.org/wiki/Canonical_ensemble) in statistical physics.
-If, for two neighboring (in the schedule sense) replicas, the values for `beta` are too different, exchanges between those replicas are unlikely to be accepted.
-If, on the other hand, all replicas are very close in `beta` space, many exchanges will be accepted, but also many, many replicas are required to bridge between the target (cold) replica and the flattest (hottest) copy, potentially wasting computing resources.
+While that family of tempering distributions is essentially arbitrary, Chainsail currently implements two tempering schemes.
+
+### Global tempering
+The most traditional and popular one tempering scheme is one where 
+
+$$
+p(x|\beta)=p(x)^\beta
+$$
+
+with $0 < \beta \leq 1$ and $p(x)$ your probability distribution of interest.
+$\beta$ is often referred to as an (inverse) temperature in analogy to the [canonical ensemble](https://en.wikipedia.org/wiki/Canonical_ensemble) in statistical physics.
+
+### Likelihood tempering
+In Bayesian inference, we have $p(x|D) \propto p(D|x) \times p(x)$.
+$p(x|D)$ is the posterior distribution and usually what a Chainsail user wants to sample.
```

Comment:
```suggestion
The posterior distribution, $p(x|D)$, is usually what a Chainsail user wants to sample.
```
It is easier to read, as in previous formulation an expression went after a full stop, after the expression. The meaning of a full stop was unclear at a first glance.